test/test-ExperimentProvider.js

```diff
@@ -1,14 +1,267 @@
 "use strict";
```

Comment:
Personally I think it's a good idea to move away from doing unit tests in `jpm` since they are fragile, hard to debug, etc., We do have a shim for `sdk/simple-prefs` already, is there a reason you need to use `sdk/preferences/service` instead?